### ISSUE TITLE
gmp and srfi-27

### ISSUE THREAD

**Issue by svenha** [manuel-serrano/bigloo#12]
I found an inconsistency: if configure detects that there is no gmp, then srfi-27 is disabled automatically. If --no-gmp is explicitly set on the configure command line, srfi-27 is not disabled. I don't know which of the two locations should be adjusted. Similarly, the output of autoconf/limbs differs: empty (autom.) vs 1 (--no-gmp)

**Comment by manuel-serrano** [comment]
Yes. You are correct. These are mistakes. I'm working at fixing them. Thanks for your report.
